databricks/koalas/indexes.py

```diff
@@ -283,7 +286,7 @@ def equals(self, other):
         >>> idx = ks.Index(['a', 'b', 'c'])
         >>> idx.name = "name"
         >>> midx = ks.MultiIndex.from_tuples([('a', 'x'), ('b', 'y'), ('c', 'z')])
-        >>> midx.names = ("nameA", "nameB")
+        >>> midx.names = ["nameA", "nameB"]
```

Comment:
I wonder if Is it okay that we can't support assigning names as a tuple that pandas can do?

```python
>>> pmidx.names = ("nameA", "nameB")
>>> pmidx
MultiIndex([('a', 'x'),
            ('b', 'y'),
            ('c', 'z')],
           names=['nameA', 'nameB'])
```

Reply:
ditto.